modules/sdk-coin-near/src/near.ts

```diff
@@ -1040,6 +1041,21 @@ export class Near extends BaseCoin {
         throw new Error('Tx total amount does not match with expected total amount field');
       }
     }
+
+    if (params.verification?.consolidationToBaseAddress) {
```

Comment:
nep141Token class is also having a verifyTransaction method. I think this will have to be added there as well